src/dart-2.0.md

```diff
@@ -59,6 +70,92 @@ Convert your code to be strong mode compliant.
 : For more information, see [Strong mode and static
   typing](#strong-mode-and-static-typing), below.
 
+## Migration tips: packages
+
+As a package owner, you need to do the following:
+
+* Follow the migration tips for the platforms that your package supports
+  (see above).
+* Make sure your package's users know how to report issues.
+* Respond quickly to issue reports.
+* If code changes aren't backward compatible,
+  update the lower SDK constraint.
+
+
+### Changes and backward compatibility
+
+If you have to change your package's code,
+**try to make it work in 1.x**, as well as 2.0.
+For example, you might be able to add type annotations
+or (if an API has been removed) to use an alternative 1.x API.
+
+If a backward-compatible change isn't possible,
+**update the lower [SDK constraint.][SDK constraints]**
+
+<aside class="alert alert-warning" markdown="1">
+  **Specify SDK constraints carefully!**
+  Incorrect lower constraints can cause problems for users of the stable SDK.
+</aside>
+
+[Test your changes][] to make sure that your package works as expected.
```

Comment:
tiny nit...

Just put the URL inline here, right?

Reply:
Done.